packages/itwin/reports-config-widget/src/widget/components/ExtractionToast.tsx

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { Text } from "@itwin/itwinui-react";
+import { ReportsConfigWidget } from "../../ReportsConfigWidget";
+
+interface ExtractionToastProps {
+  iModelName: string;
+  odataFeedUrl?: string;
+}
+
+export const SuccessfulExtractionToast = ({ iModelName, odataFeedUrl }: ExtractionToastProps) => {
+  return (
+    <div>
+      <Text>{ReportsConfigWidget.localization.getLocalizedString("ReportsConfigWidget:ExtractionSuccess")}{iModelName} </Text>
+      <a href="#" onClick={async () => { await navigator.clipboard.writeText(odataFeedUrl!); }}>{ReportsConfigWidget.localization.getLocalizedString("ReportsConfigWidget:CopyODataUrl")}</a>
```

Comment:
Does `#` appear in the url when you click on this link?

Reply:
`e.preventDefault();` has been added. The `#` no longer appears in the link